### ISSUE TITLE
Changing hash causes API call

### ISSUE THREAD

**Issue by Swamii** [5monkeys/django-bananas.js#1]
Say if I have a `read.js`-file where I use the url hash to store the currently selected tab. Every time the user changes tab, the hash is updated like this:
```js
this.context.router.route({
  ...route,
  hash: `tab-${activeTab}
});
```
Then the read api call is executed. It would be great if either that doesn't happen or it is configurable somehow. Maybe a param can be added to `route()`?

**Comment by lydell** [comment]
I believe this was fixed in v1.2.5.

**Comment by Swamii** [comment]
> I believe this was fixed in v1.2.5.

Indeed it was. I'm closing this.
